queries.sql

```diff
@@ -188,4 +188,124 @@ FROM animals A
 JOIN owners O
 ON A.owner_id = O.id
 GROUP BY O.full_name
-ORDER BY animal DESC;
\ No newline at end of file
+ORDER BY animal DESC;
+
+/*MANY-TO-MANY QUERIES*/
+
+SELECT
+A.name AS animal_name,
+vets.name AS vet_name,
+visits.date_of_visit
+FROM visits
+JOIN animals A
+ON A.id = visits.animal_id
+JOIN vets
+ON vets.id = visits.vet_id
+WHERE vets.name = 'William Tatcher'
+ORDER BY visits.date_of_visit DESC;
+
+SELECT DISTINCT
+vets.name,
+COUNT(A.name) AS animals_attended
+FROM visits
+JOIN animals A
+ON A.id = visits.animal_id
+JOIN vets
+ON vets.id = visits.vet_id
+GROUP BY vets.name;
+
+SELECT
+vets.name AS vet_name,
+species.name AS specialization
+FROM specializations S
+JOIN species
+ON species.id = S.species_id
+RIGHT JOIN vets
+ON vets.id = S.vet_id;
```

Comment:
- [ ] Great job! 🌟 However, I'd like to make it easier to understand the specific analytical question that this query is addressing. 🤔💡 Let's clearly indicate what problem or question we're trying to solve with this query. 🚀🔍
- [ ] Please adjust your screenshots 📸 and, if possible, add a note to the PR description 📝 indicating which query the screenshot corresponds to. 🤓🔍 #QueryClarification